### PR TITLE
Criteo audience doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

> Describe the changes you are proposing in this PR.

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/criteo-audience.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add the source. Then, from the list of destinations, select **Criteo Audience**.
+2. Assign a name to the destination and click **Continue**.
+
+## Connection settings
+
+To successfully configure Criteo Audience as a destination, you need to configure the following settings:
+
+- **Create Account**: Click **Create Account** > **Connect with Criteo Audience** and enter your login credentials for Criteo Audeince.
+- **Audience Id**: Enter the ID of the audience for which you want to sync the data. Refer to the [Criteo Audience](https://developers.criteo.com/marketing-solutions/docs/audience#creating-a-new-audience) documentation for more information on obtaining the audience ID.
+- **Audience Type**: Enter the type of audience for which you want to add or remove the users.
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/criteo-audience.mdx:31]:
```suggestion
- **Audience Type**: Enter the type of audience by which you want to add or remove the users.
```

File: docs/destinations/streaming-destinations/criteo-audience.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  }
+}
+```
+
+In the above snippet, you can pass the `identifierType` as:
+- `email`: Email Address
+- `madid`: Mobile Ad Identifier
+- `identityLink`: LiveRamp Identity Link
+- `gum`: Criteo GUM cookie identifier
+
```
Comment (anantjain45823) [inline docs/destinations/streaming-destinations/criteo-audience.mdx:61]: Can we please add a note somewhere that users must add the following redirect URLn their Application Details:
https://app.rudderstack.com/auth/destination/criteo_audience/callback

Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/criteo-audience.mdx:65]: added

**Review by anantjain45823 — APPROVED** [review]

**Review by ujjwal-ab — APPROVED** [review]

File: docs/destinations/streaming-destinations/criteo-audience.mdx [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+title: "Criteo Audience"
+description: Send your event data from RudderStack to Criteo Audience.
+---
+
+[Criteo](https://www.criteo.com/) is an online display advertising platform and its [Audience API](https://developers.criteo.com/marketing-solutions/docs/audiences) lets you manage your contact lists by creating, deleting, and updating audience data, which is used for your advertising campaigns.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/criteo-audience.mdx:6]:
```suggestion
[Criteo](https://www.criteo.com/) is an online display advertising platform. Its [Audience API](https://developers.criteo.com/marketing-solutions/docs/audiences) lets you manage your contact lists by creating, deleting, and updating audience data, which you can use for your advertising campaigns.
```

File: docs/destinations/streaming-destinations/criteo-audience.mdx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+Once you have confirmed that the source platform supports sending events to Criteo Audience, follow these steps:
+
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add the source. Then, from the list of destinations, select **Criteo Audience**.
+2. Assign a name to the destination and click **Continue**.
+
+## Connection settings
+
+To successfully configure Criteo Audience as a destination, you need to configure the following settings:
+
+- **Create Account**: Click **Create Account** > **Connect with Criteo Audience** and enter your login credentials for Criteo Audeince.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/criteo-audience.mdx:28]:
```suggestion
- **Create Account**: Click **Create Account** > **Connect with Criteo Audience** and enter your login credentials for Criteo Audience.
```

File: docs/destinations/streaming-destinations/criteo-audience.mdx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+- **Audience Id**: Enter the ID of the audience for which you want to sync the data. Refer to the [Criteo Audience](https://developers.criteo.com/marketing-solutions/docs/audience#creating-a-new-audience) documentation for more information on obtaining the audience ID.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/criteo-audience.mdx:29]:
```suggestion
- **Audience Id**: Enter the audience ID for which you want to sync the data. Refer to the [Criteo Audience documentation](https://developers.criteo.com/marketing-solutions/docs/audience#creating-a-new-audience) for more information on obtaining the audience ID.
```

File: docs/destinations/streaming-destinations/criteo-audience.mdx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+- **Audience Type**: Select the type of audience from the dropdown for which you want to add or remove the users. If you select `gum`, then **Gum Caller Id** is a required field. Refer to the [Criteo documentation](https://developers.criteo.com/marketing-solutions/docs/contact-list#adding-and-removing-users-in-an-audience) for more information on different audience types.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/criteo-audience.mdx:30]:
```suggestion
- **Audience Type**: Select the audience type from the dropdown for which you want to add or remove the users. If you select `gum`, then **Gum Caller Id** is a required field. Refer to the [Criteo documentation](https://developers.criteo.com/marketing-solutions/docs/contact-list#adding-and-removing-users-in-an-audience) for more information on different audience types.
```

File: docs/destinations/streaming-destinations/criteo-audience.mdx [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+        {
+          "madid": "sample_madid_1"
+        }
+      ]
+    }
+  }
+}
+```
+
+Criteo's API implements [rate limits](https://developers.criteo.com/marketing-solutions/docs/rate-limits) to limit the number of requests you're able to perform on any API endpoint per minute.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/criteo-audience.mdx:74]:
```suggestion
<div class="warningBlock">
Criteo's API implements [rate limits](https://developers.criteo.com/marketing-solutions/docs/rate-limits) to limit the number of requests you're able to perform on any API endpoint per minute.
</div>

```

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**